code/src/index.css

```diff
@@ -1,13 +1,410 @@
+    color: rgb(235, 219, 173);
+    border-bottom: 5px solid  rgb(235, 219, 173);
+    font-size: 20px; 
+  }
+
+  textarea:focus, input:focus {
+    color: rgb(235, 219, 173);
+}
+
+  .user-input::placeholder{
+    color: rgb(235, 219, 173);
+  }
+
+  .star-shop {
+    
+    font-size: 30px;
+    padding-bottom: 5px;
+    line-height: 36px;
+  }
+
+  .star-shop-select {
+    border:none;
+    background-color: rgb(235, 219, 173);
+    color: black;
+    border-bottom: 5px solid  rgb(235, 219, 173);
+    margin-bottom: 20px;
+  }
+
+  .star-shop-select-option {
+    background-color: white;
+  }
+
+  .star{
+    font-family: 'Permanent Marker', cursive;
+  }
+
+  .label {
+    padding-top:5px;
+    color:rgb(235, 219, 173);
+  }
+
+  .check-h1 {
+    padding-bottom:10px
+  }
+
+  .list {
+    margin:7px;
+  }  
+
+  .survey-completed-h1 {
+    margin-bottom: 20px;
+  }
+
+  /* ------------------tablet section---------------------- */
+@media (min-width: 667px) and (max-width: 1024px) {
+  .body-section {
+    display: grid;
+    grid-template-columns: 2rem 1fr 1fr 1fr 1fr 2rem;
+    grid-template-rows: 2rem 1fr 1fr 1fr 2rem;
+    height: 100vh;
+    flex-direction: column;
+    overflow-y: scroll;
+    top: 0; right: 0; bottom: 0; left: 0;
+  }
+
+  .first-page-h1 {
+    font-size: 40px;
+    margin-bottom: 10%;
+  }
+
+  .first-page-p {
+    font-size: 25px;
+    line-height: 50px;
+  }
+   
+  .bottom {
+    margin-bottom: 7%;
+  }
+
+  .subscribe-button {
+    padding: 3px 15px 7px 15px;
+    font-size: 30px;
+    font-weight: bolder;
+    font-family: 'Permanent Marker', cursive;
+    border-color: bisque;
+    border-radius: 45%;
+    transition: all 0.1s ease-out 0s;
+  } 
+
+  .question-section {
+    grid-column-start: 3;
+
+  }
+
+  .previous-button {
+    grid-column-start: 2;
+    grid-row-start: 4; 
+    margin: 0;
+    padding: 21px;
+    justify-self: start;
+  }
+  
+  .next-button {
+    grid-column-start: 2;
+    grid-row-start: 4;
+    margin: 0; 
+    padding: 21px;
+    justify-self: center;
+  }
+
+  .clear-button {
+    padding: 17.5px;
+    grid-column-start: 5;
+    font-size: 26px;
+  }
+
+  .user-p {
+    padding-bottom:60px;
+    font-size: 50px;
+    line-height: 50px;
+  }
+
+  .user-input {
+    padding: 20px;
+    font-size: 20px;
+  }
+
+  .ok-button {
+    margin: 15px;
+    padding: 10px;
+  }
 }
+
+/* ------------------computer section---------------------*/
+@media (min-width: 1025px) {  
+  .first-page-h1 {
+    font-size: 40px;
+    margin-bottom: 5%;
+  }
+
+  .first-page-p {
+    font-size: 25px;
+    line-height: 50px;
+  }
+  
+  .bottom {
+    margin-bottom: 7%;
+  }
+
+  .subscribe-button {
+    padding: 3px 15px 7px 15px;
+    font-size: 30px;
+    font-weight: bolder;
+    font-family: 'Permanent Marker', cursive;
+    border-color: bisque;
+    border-radius: 45%;
+    transition: all 0.1s ease-out 0s;
+  } 
+
+  .subscribe-button:hover {
+    font-size: 35px;
+    transition: all 0.1s ease-out 0s;
```

Comment:
Suggestion to try and see if the `transform: scale()` property works instead of increasing the fontsize, I believe this would help to make the other parts of the page not to move when hovering. Here's a good article on the scale property: https://css-tricks.com/almanac/properties/s/scale/